src/main/java/core/basesyntax/SqrtOperation.java

```diff
@@ -8,6 +8,10 @@
  */
 public class SqrtOperation {
     public double sqrt(double number) {
-        return 0.0;
+        if (number < 0) {
+            throw new
+                java.lang.IllegalArgumentException("Expected non-negative number, got " + number);
```

Comment:
```suggestion
                throw new IllegalArgumentException("Expected non-negative number, got " + number);
```
package names are redundant here

Reply:
done